Podcast/BookmarkViewController.swift

```diff
@@ -30,20 +30,23 @@ class BookmarkViewController: UIViewController, UITableViewDelegate, UITableView
         bookmarkTableView.backgroundColor = .clear
         bookmarkTableView.separatorStyle = .none
         bookmarkTableView.showsVerticalScrollIndicator = false
-        bookmarkTableView.register(CardTableViewCell.self, forCellReuseIdentifier: "CardTableViewCellIdentifier")
+        bookmarkTableView.register(BookmarkTableViewCell.self, forCellReuseIdentifier: "BookmarkTableViewCellIdentifier")
         view.addSubview(bookmarkTableView)
-        bookmarkTableView.rowHeight = CardTableViewCell.height
+        bookmarkTableView.rowHeight = BookmarkTableViewCell.height
         bookmarkTableView.reloadData()
         
-        cards = fetchCards()
+        episodes = fetchEpisodes()
```

Comment:
you can probably assign episodes inside of the fetchEpisodes func so you don't have to return anything (especially since its going to be async)